charts/budibase/templates/app-service-deployment.yaml

```diff
@@ -98,10 +98,6 @@ spec:
           value: http://worker-service:{{ .Values.services.worker.port }}
         - name: PLATFORM_URL
           value: {{ .Values.globals.platformUrl | quote }}
-        - name: USE_QUOTAS
```

Comment:
Don't forget to bump the version in `Chart.yaml` to cut a new release of the helm chart when this goes out 👍

Reply:
👍